roles/parse_operator_bundle/tasks/main.yml

```diff
@@ -17,6 +17,18 @@
     set_fact:
       skopeo_inspect_json: "{{ skopeo_inspect_result.stdout | from_json }}"
 
+  - name: "Verify required labels"
+    block:
+    - set_fact:
+        failed_label: ""
+    - set_fact:
+        failed_label: "{{ item }}"
```

Comment:
Suggestion: we may want to aggregate all missing labels here in case more than one is missing.